### ISSUE TITLE
Main door key missing

### ISSUE THREAD

**Issue by aizenreigns** [fiskybusiness/phasmophobia#32]
For some reason I am unable to play the game, I'm trying to test if the mod works but the main door key on all of the maps dont spawn or is not present on the table. Do i need to fill the lobby first? Everything else spawns including the items on the shelves
![aaa](https://user-images.githubusercontent.com/72643583/95647626-82ab3d80-0b03-11eb-9ba6-e74631987980.png)

**Comment by aizenreigns** [comment]
![image](https://user-images.githubusercontent.com/72643583/95647655-c1d98e80-0b03-11eb-8d05-5c4704eba918.png)

**Comment by Zombieblowup** [comment]
Now this is odd! Config and version? The key should spawn as soon as all the players join or almost instant if solo.

**Comment by myrukun** [comment]
yeah this happened to me but i just remade the lobby and made the players less
